app/views/layouts/application.html.erb

```diff
@@ -99,5 +99,7 @@
         })();
       </script>
     <% end %>
+
+    <%= render "search/whats_cwic_modal" %>
```

Comment:
I'm moving this, but for future reference, this sort of thing doesn't belong in application.html.erb. Putting it here will cause it to be included in all the accounts pages, data access, etc, when it's only needed in the granules list for some collections